bin/accumulo-testing

```diff
@@ -223,6 +224,12 @@ ci-mapred)
   build_shade_jar
   "$HADOOP_PREFIX"/bin/yarn jar "$at_shaded_jar" "$ci_main" "$at_props"
   ;;
+rw-list)
+  echo "Available randomwalk modules:"
+  for x in $at_home/core/src/main/resources/randomwalk/modules/*.xml ; do
```

Comment:
For runtime, does mvn package put this in a diff place?

Reply:
The module class looks on the classpath for these files.